src/background.js

```diff
@@ -31,9 +31,11 @@ chrome.tabs.onAttached.addListener((tabId, data) => {
   // so we can do the same to the next one to open.
   chrome.tabs.get(tabId, tab => {
     if (tab.url.startsWith(URLS.ownOrigin + 'edit.html')) {
-      chrome.windows.get(tab.windowId, {populate: true}, win => {
+      chrome.windows.get(tab.windowId, {
+        populate: true
```

Comment:
Hehe, I might have used "Atom beautify" a couple of times, so yeah it was literally a robotic expansion 😛. I'll return these to their previous state.